### ISSUE TITLE
Installing from PyPI section assumes it will be an adafruit build

### ISSUE THREAD

**Issue by lesamouraipourpre** [adafruit/cookiecutter-adafruit-circuitpython#130]
The *Installing from PyPI* section in README.rst assumes that it will be an **adafruit** cookiecutter run and doesn't allow for a **community** run. eg.
```text
.. code-block:: shell

    pip3 install adafruit-circuitpython-{{ pypi_name }}
```

**Comment by FoamyGuy** [comment]
I just noticed this as well working on circuitpython org repo. The `from PyPi` link location in the README near this also has adafruit embedded by default

**Comment by kattni** [comment]
Fixed in #133
